diff --git a/node_modules/realm/RealmJS.podspec b/node_modules/realm/RealmJS.podspec
index 54cacad..088ef2a 100644
--- a/node_modules/realm/RealmJS.podspec
+++ b/node_modules/realm/RealmJS.podspec
@@ -47,7 +47,7 @@ Pod::Spec.new do |s|
   # Note: It leaves a lock file, ensuring it will only download the archive once.
   s.prepare_command        = './scripts/xcode-download-realm.sh ./scripts'
   s.script_phase           = { :name => 'Download Realm Core & Sync',
-                               :script => '${PODS_TARGET_SRCROOT}/scripts/xcode-download-realm.sh ${PODS_TARGET_SRCROOT}/scripts',
+                               :script => 'echo "Using Node.js $(node --version)" && node "${PODS_TARGET_SRCROOT}/scripts/download-realm.js" ios --sync',
                                :execution_position => :before_compile }
 
   s.source_files           = 'src/*.cpp',
